projects/ccf-rui/src/themes/common/_theming.scss

```diff
@@ -27,6 +29,8 @@
   @include rightSidebar.theme($theme);
   @include blockSizeInput.theme($theme);
   @include slicesInput.theme($theme);
+  @include sideSelector.theme($theme);
```

Comment:
Minor thing. Please move down to after organ-selector. I am trying to keep this file a bit tidier by grouping things together based on ui location.